backend/spec/models/group_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Group do
+  subject(:group1) { described_class.create(group_params) }
+
+  let(:current_user) { create(:user) }
+  let(:name) { 'SAWQ' }
```

Comment:
make name valid

Reply:
> make name valid

It's not make sense
There i check minimum symbols in my model